scripts/run_e2e_tests.py

```diff
@@ -127,6 +127,29 @@ def is_oppia_server_already_running() -> bool:
     return False
 
 
+def run_ng_compilation() -> None:
```

Comment:
Why is this code repeated in multiple places? Please use common.py if you need to reuse stuff.

Reply:
A similar block seems to be added to scripts/run_lighthouse_tests.py

Reply:
Done!